tests/integration_tests/test_tflite_detect.py

```diff
@@ -63,3 +45,40 @@ def test_pet_detection(self):
             assert (
                 found_pet
             ), f"No cat/dog detected with >50% confidence in {image_file} predictions: {predictions}"
+
+    def test_non_pet_detection(self):
+        # Initialize detector
+        detector = TFLiteDetect()
+
+        # Get all images from test directory
+        test_image_dir = "src/basic_bot/test_helpers/data/not_pet_images"
+        image_files = [
+            f
+            for f in os.listdir(test_image_dir)
+            if f.endswith((".jpg", ".jpeg", ".png"))
+        ]
+
+        # Test each image
+        for image_file in image_files:
+            image_path = os.path.join(test_image_dir, image_file)
+            img = cv2.imread(image_path)
+
+            # Get predictions
+            predictions = detector.get_prediction(img)
+
+            if len(predictions) <= 0:
+                print(f"No predictions for {image_file}")
+
+            # Check if any prediction is cat/dog with >50% confidence
+            found_pet = False
+            for pred in predictions:
+                if (
+                    pred["classification"].lower() in ["cat", "dog"]
+                    and pred["confidence"] > 0.5
+                ):
+                    found_pet = True
+                    break
+
+            assert (
+                not found_pet
+            ), f"No cat/dog detected with >50% confidence in {image_file} predictions: {predictions}"
```

Comment:
This output statement is wrong. s/b f"Should not have found cat or dog in {image_file} predictions: {predictions}"

Also update the test_pet_detection test above this test out put to be more BDD-like